test/image/export_test.js

```diff
@@ -1,106 +1,89 @@
 var fs = require('fs');
-var path = require('path');
 
+var getMockList = require('./assets/get_mock_list');
+var getRequestOpts = require('./assets/get_image_request_options');
+var getImagePaths = require('./assets/get_image_paths');
+
+// packages inside the image server docker
 var request = require('request');
 var test = require('tape');
 
-var constants = require('../../tasks/util/constants');
-var getOptions = require('../../tasks/util/get_image_request_options');
+// image formats to test
+// N.B. 'png' is tested in `npm run test-image, no need to duplicate here
+// TODO figure why 'jpeg' and 'webp' lead to errors
```

Comment:
Yes. It's because the image export main assumes that `Plotly.Color.combine` is exposed to blend the bgcolor with white for formats that don't support transparent backgrounds. As `Color` got removed from `Plotly` before the FOSS release, `jpg` and `webp` exports fail with the default plotly.js `Plotly`. Note: they still work work on plot.ly.